larpandoracontent/LArControlFlow/StitchingCosmicRayMergingTool.cc

```diff
@@ -524,6 +524,49 @@ void StitchingCosmicRayMergingTool::StitchPfos(const MasterAlgorithm *const pAlg
         for (const ParticleFlowObject *const pPfo : pfoList) pfoVector.push_back(pPfo);
         std::sort(pfoVector.begin(), pfoVector.end(), LArPfoHelper::SortByNHits);
 
+        std::pair<const LArTPC*, const LArTPC*> stitchedLArTPCs;
```

Comment:
Do make sure the values in the pair are explicitly initialised (you rely on this below), e.g. via the constructor.

Typically make this a typedef, here probably a private typedef defined in the class prototype. Can here then make an instance via the more self-describing e.g. `LArTPCPair stitchedLArTPCs(nullptr, nullptr);`